docs/changelog.rst

```diff
@@ -7,6 +7,11 @@ Changelog
 See the `Releases page`_ on GitHub for a complete list of commits that are
 included in each version.
 
+2.12.0 (2024-Dec-XX)
+--------------------
+
+- Remove the ``assert_error`` plugin method.
```

Comment:
Doing diligence: is there a replacement or fallback?

Reply:
No, this is removing it because it was a breaking change.

Reply:
Hmm though maybe:

```suggestion
- Remove the ``assert_error`` pytest plugin method. Use ``capsys`` to check for errors.
```

Reply:
The tense makes the grammar a bit ambiguous.

```suggestion
- Remove the ``assert_error`` pytest method. For checking errors, we recommend
  using ``capsys`` instead.
```

Reply:
Done!